libraries/ipc/ipc.c

```diff
@@ -176,8 +176,9 @@ return_type ipc_service_connection_request(ipc_structure_type *ipc_structure)
     message_parameter_type message_parameter;
     uint8_t data[100];
 
-    // FIXME: Make it possible to specify the size of the mailbox. For now, we just set it to one meg and hope it's enough.
-    if (system_call_mailbox_create(&ipc_structure->input_mailbox_id, 1 * MB, PROCESS_ID_NONE,
+    // FIXME: Make it possible to specify the size of the mailbox. For now, we just set it to a
+    // fixed size and hope it's enough.
+    if (system_call_mailbox_create(&ipc_structure->input_mailbox_id, 64 * KB, PROCESS_ID_NONE,
```

Comment:
The careful reader might notice that the mailbox size is decreased greatly here, because the whole buffer for it is now preallocated (and it's much worse to waste `n * 1 MiB` vs `n * 64 KiB` of mostly-unused memory). We might have to make this configurable in the IPC library at some point, but for now, "64 KiB ought to be enough for anybody". 😇